src/i_joystick.h

```diff
@@ -93,7 +93,8 @@ typedef enum
     GAMEPAD_BUTTON_PADDLE4 = SDL_CONTROLLER_BUTTON_PADDLE4,
     GAMEPAD_BUTTON_TOUCHPAD = SDL_CONTROLLER_BUTTON_TOUCHPAD,
     GAMEPAD_BUTTON_TRIGGERLEFT,
-    GAMEPAD_BUTTON_TRIGGERRIGHT
+    GAMEPAD_BUTTON_TRIGGERRIGHT,
+    GAMEPAD_BUTTON_MAX,
```

Comment:
Trailing comma, does this work on every C compiler ever?

Reply:
I am not aware of any issue with trailing commas in C. For many people (myself included!) they are considered to be best practice. Indeed, I see them being used in many places elsewhere in chocolate-doom.

Reply:
Trailing comma in `enum` appears to be a C99 addition.

Reply:
Ah, I didn't realize the `enum` trailing comma only came in with C99. Array initializer trailing comma is in K&R 2nd edition (C89) so I think those should be fine just about everywhere.

I'll take this specific trailing comma out.